### PR TITLE
Move custom DB scripts to separate files

### PR DESCRIPTION
**Pull request by joshcanhelp**
### Changes

This is the first PR in a process to improve the user migration setup process. 

- Move custom DB scripts from PHP string to separate JS file
- Add method to get script file contents with a name
- Updated tests for minor format changes
- Update version number to `3.9.0-beta`

### Testing

No new tests added, previous tests updated to new format. 

### Checklist

* [x] All existing and new tests complete without errors
* [x] All code quality tools/guidelines in the [Contribution guide](CONTRIBUTION.md) have been run/followed
* [x] All active GitHub CI checks have passed

### CODE REVIEW

File: lib/WP_Auth0_DBManager.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 		$sso             = $options->get( 'sso' );
```
Comment (joshcanhelp) [inline lib/WP_Auth0_DBManager.php]: Just comments in this file ... I was researching the DB version and wanted to persist what I found.

File: WP_Auth0.php [hunk 2]
```diff
@@ -2,13 +2,13 @@
 /**
  * Plugin Name: Login by Auth0
  * Description: Login by Auth0 provides improved username/password login, Passwordless login, Social login and Single Sign On for all your sites.
- * Version: 3.8.1
+ * Version: 3.9.0-beta
```
Comment (joshcanhelp) [inline WP_Auth0.php]: First change after a release needs to bump the version number in `master` in a way that can be updated when the official release comes out.

File: lib/WP_Auth0_CustomDBLib.php [hunk 3]
```diff
@@ -1,7 +1,41 @@
...
+	/**
+	 * Get JS to use in the custom database script.
+	 *
+	 * @param string $name  - Database script name.
+	 * @param string $token - Migration token.
+	 *
+	 * @return bool|string
+	 */
+	public static function get_script( $name, $token ) {
+		$script = (string) file_get_contents( WPA0_PLUGIN_DIR . 'lib/scripts-js/db-' . $name . '.js' );
```
Comment (cocojoe) [inline lib/WP_Auth0_CustomDBLib.php]: Whenever I see `file_get_contents` followed by a variable. 👀

Comment (joshcanhelp) [reply lib/WP_Auth0_CustomDBLib.php]: https://secure.php.net/manual/en/function.file-get-contents.php

> This function is similar to file(), except that file_get_contents() returns the file in a string, starting at the specified offset up to maxlen bytes. On failure, file_get_contents() will return FALSE.

Worst that happens is a PHP warning if the file does not exist.

**Pull request merged**